src/PIL/MpoImagePlugin.py

```diff
@@ -80,6 +80,8 @@ def seek(self, frame):
             ("jpeg", (0, 0) + self.size, self.offset, (self.mode, ""))
         ]
         self.__frame = frame
+        self.fp.seek(0)
+        JpegImagePlugin.JpegImageFile._open(self)
```

Comment:
Would `super(MpoImageFile, self)._open()` not be better?

Reply:
@radarhere - I was simply reusing the existing syntax used above (https://github.com/jmetz/Pillow/blob/ebb78aecb94227e00e259931df70581f7f5b56cf/src/PIL/MpoImagePlugin.py#L46 ) . 
Could switch to `super` version though (should I change both?) if that's preferred.

Reply:
Ah. Fair enough then! Well, we'll see how #3149 goes

Reply:
Wait, I've realised there is already discussion about this question in #2198. Ignore my comment, your code is fine.